xrpl/models/transactions/did_set.py

```diff
@@ -12,7 +12,9 @@
 from xrpl.models.transactions.types import TransactionType
 from xrpl.models.utils import require_kwargs_on_init
 
-HEX_REGEX: Final[Pattern[str]] = re.compile("[a-fA-F0-9]+")
+# DID LedgerObject fields are removed with an empty string as input.
+# Hence, empty strings need to be supported in the regex
```

Comment:
Code comments aren't useful if they're only relevant to the PR. This comment won't make any sense later on when looking at the code.

Reply:
I can remove this comment. But I believe it's useful.

It tells future developers why the regex shouldn't support empty strings.

Reply:
"... regex should support empty strings"

Reply:
I think that should be discussed in the docs for each of the parameters, which should be copied from the docs here: https://xrpl.org/docs/references/protocol/transactions/types/didset

Reply:
I agree. I've updated it in [e96a8ce](https://github.com/XRPLF/xrpl-py/pull/717/commits/e96a8cec419b1473086bfa76b0d996bdbb42b038)